numpy/core/arrayprint.py

```diff
@@ -1346,7 +1346,7 @@ def dtype_is_implied(dtype):
     return dtype.type in _typelessdata
 
 
-def dtype_short_repr(dtype):
+def dtype_short_repr(_dtype):
```

Comment:
Trailing underscore would probably be better here and consistent with other usage in this module.

Reply:
fixed by removing the import